src/components/shared/FormElements/Checkboxes/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import PropTypes from 'prop-types';
+import dompurify from 'dompurify';
+
+// Import styling
+import s from './Checkbox.module.scss';
+
+const { sanitize } = dompurify;
+
+const Checkbox = ({ name, fieldValidation, text, value, checked, onChange }) => {
+  return (
+    <>
+      <label>
+        <input
+          className={`${s.checkbox}`}
+          name={name}
+          type="checkbox"
+          ref={fieldValidation}
+          value={value}
+          checked={checked}
+          onChange={onChange}
+        />
+        <div
+          className={`wmnds-btn wmnds-btn--secondary wmnds-btn--block ${s.button} `}
+          dangerouslySetInnerHTML={{
+            __html: sanitize(text),
+          }}
+        />
+      </label>
```

Comment:
I would use the label as the button instead of a div here and add an `htmlFor` to link it to the input e.g. 
```suggestion
      <div>
        <input
          id={id}
          className={`${s.checkbox}`}
          name={name}
          type="checkbox"
          ref={fieldValidation}
          value={value}
          checked={checked}
          onChange={onChange}
        />
        <label
          className={`wmnds-btn wmnds-btn--secondary wmnds-btn--block ${s.button} `}
          htmlFor={id}
          dangerouslySetInnerHTML={{
            __html: sanitize(text),
          }}
        />
      </div>
```
Then the es-lint won't need to be disabled.